package.json

```diff
@@ -32,6 +32,7 @@
     "@types/nodemailer": "^4.6.8",
     "danger": "^4.0.2",
     "danger-plugin-digitalcitizenship": "^0.3.1",
+    "dotenv": "^8.2.0",
```

Comment:
a question, does dotenv configured in jest.config.js read `env.example` by default ?
I know it reads `.env` by default but maybe env.example is included as well.

Reply:
I think it's not, we might specify it explicitly

Reply:
done :)

Reply:
Yes, the correct file name is `.env`. 
There is a comment in the first line of this file that indicates to rename, the idea is to indicate that it is just a test file.